htdocs/mrp/class/mo.class.php

```diff
@@ -1980,6 +1980,75 @@ public function getMoParent()
 		}
 	}
 
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
+	/**
+	 *      Load indicators for dashboard (this->nbtodo and this->nbtodolate)
+	 *
+	 *      @param	User	$user   		Object user
+	 *      @return WorkboardResponse|int 	Return integer <0 if KO, WorkboardResponse if OK
+	 */
+	public function load_board($user)
+	{
+		// phpcs:enable
+		global $conf, $langs;
+		if ($user->socid) {
+			return -1; // Protection pour éviter appel par utilisateur externe
+		}
+
+		$now = dol_now();
+
+		$sql = "SELECT rowid, date_end_planned FROM ".$this->db->prefix()."mrp_mo";
+		$sql .= " WHERE status IN (" . self::STATUS_VALIDATED . ", " . self::STATUS_INPROGRESS .")"; // 1 = Ouvert, 2 = En cours
+		$sql .= " AND entity IN (".getEntity('mrp_mo').")";
+
+		$resql = $this->db->query($sql);
+		if ($resql) {
+			$langs->load("mrp");
+			$response = new WorkboardResponse();
+			$warning_delay = $conf->mrp->progress->warning_delay ;
+			$response->warning_delay = $warning_delay / 86400;
+			$response->label = $langs->trans("MOProgress");
+			$response->labelShort = $langs->trans("MOProgress");
+			$response->url = DOL_URL_ROOT.'/mrp/mo_list.php?search_status=-2';
+			$response->img = img_object('', "mrp");
+
+
+			while ($obj = $this->db->fetch_object($resql)) {
+				$response->nbtodo++;
+
+				if (!empty($obj->date_end_planned)) {
+					$date_end_planned = $this->db->jdate($obj->date_end_planned);
+					if ($now > ($date_end_planned + $warning_delay)) {
+						$response->nbtodolate++;
+						$response->url_late = DOL_URL_ROOT.'/mrp/mo_list.php?search_status=-2&search_option=late';
+					}
+				}
+			}
+
+			return $response;
+		} else {
+			dol_print_error($this->db);
+			$this->error = $this->db->error();
+			return -1;
+		}
+	}
+
+	/**
+	 * Is the manufactured delayed?
+	 *
+	 * @return bool
+	 */
+	public function hasDelay()
+	{
+		global $conf;
+
+		if ($this->status != Mo::STATUS_VALIDATED && $this->status != Mo::STATUS_INPROGRESS) {
+			return false;
+		}
+		return dol_now() > ($this->date_end_planned + $conf->mrp->progress->warning_delay) ? true : false;
```

Comment:
Is > before the ? or after. This may vary on php version.
So add parenthesis to show what you wan first.